packages/browser/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -33,9 +33,12 @@ import {
   IRedirector,
   IStorageUtility,
 } from "@inrupt/solid-client-authn-core";
-import URL from "url-parse";
 import { injectable, inject } from "tsyringe";
-import IJoseUtility from "../../../jose/IJoseUtility";
+import { default as Oidc, OidcClient, SigninRequest } from "oidc-client";
+
+// We set the logging details for all of oidc-client here
+Oidc.Log.logger = console;
+Oidc.Log.level = Oidc.Log.DEBUG;
```

Comment:
I don't think we want to set this for our consumers. Either we expose this configurability to the developer, or we keep it at the default.

Reply:
Yeah absolutely, I'd already removed this locally.